mgr.go

```diff
@@ -224,27 +225,59 @@ func newSysboxMgr(ctx *cli.Context) (*SysboxMgr, error) {
 		return nil, fmt.Errorf("failed to compute kernel-module mounts: %v", err)
 	}
 
+	useIDMapping, err := getIDMappingSupport(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to check kernel ID-mapping support: %v", err)
+	}
+
+	useIDMappingOnOvfs := useIDMapping
```

Comment:
I did it because if `useIDMapping = false`, I want `useIDMappingOnOvfs` to be false too (i.e., the former is a precondition for the latter). On the other hand, if `useIDMaping` is true, then the `if` statement in L235 below goes into a further check to see if `useIDMappingOnOvfs` should be true or false.